protobuf-format.md

```diff
@@ -138,12 +137,39 @@ Transports that support content identification MUST use the following designatio
    application/cloudevents+protobuf
 ```
 
-## 5. Examples
+## 5. Batch Format
+
+Batch format allows for a set of CloudEvents to be represented.
+
+Although the _protobuf batch format_ builds on the _protobuf format_ it is considered
+seperate, that is to say that support of _protobuf format_ does not indicate support
+of the batch representation. The batch format MUST only be used where supported.
+
+### 5.1 Envelope
+
+The enveloping container is a _CloudEventBatch_ protobuf message containing a
+repeating set of _CloudEvent_ message(s):
+
+```proto
+message CloudEventBatch {
+  repeated CloudEvent event = 1;
```

Comment:
```suggestion
  repeated CloudEvent events = 1;
```

Explanation: while Google style used to suggest using a singular name even for repeated fields, these days (certainly for public protos) we use plural names.